examples/tagmap/webpack.config.js

```diff
@@ -48,4 +48,4 @@ module.exports = {
 
 // DELETE THIS LINE WHEN COPYING THIS EXAMPLE FOLDER OUTSIDE OF DECK.GL
 // It enables bundling against src in this repo rather than installed deck.gl module
-// module.exports = require('../webpack.config.local')(module.exports);
+module.exports = require('../webpack.config.local')(module.exports);
```

Comment:
Nit: There is usually an `env` check on this line to make this work outside of repo?